test/test-notification-model-updates.js

```diff
@@ -0,0 +1,382 @@
+var ServiceManager = require('../index').ServiceManager;
+var async = require('async');
+var meshServer = require('../index').meshServer;
+var os = require('os');
+var test = require('tap').test;
+
+test('Test notifications', function(t) {
+  var app = meshServer(new ServiceManager());
```

Comment:
I would like to keep this explicit. The default implementation should only be used during tests. Any real usecase would need to pass in a real implementation.